apps/web/pages/api/proofs/cb1/index.ts

```diff
@@ -37,6 +37,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
   if (req.method !== 'GET') {
     return res.status(405).json({ error: 'method not allowed' });
   }
+  logger.info('cb1 proofs request', { query: req.query });
```

Comment:
leftover?

Reply:
I want to keep for one deploy, to be 100% sure things are working in prod. Once we. know this works for sure I'll remove